020-LinuxFundamentals/Student/Challenge-12.md

```diff
@@ -0,0 +1,40 @@
+# Challenge 12 - Setting up a webserver 
+
+[< Previous Challenge](./Challenge-11.md) - **[Home](../README.md)** 
+
+## Description
+
+In this challenge we will setting up a webserver and deploy a simple php application into it. As a plus, you can add SSL to ensure security requirements.
+
+- Download the sample application [from here](./resources/simple-php-app.tar.gz) to your home directory
+- Extract the content of simple-php-app.tar.gz on our home directory
+- Install nginx-core
+- Install php7.4-fpm
+- Configure Nginx
+
+## Success Criteria
+
+1. Make sure you have the file simple-php-app-tar.gz within your `~`
+2. Show the content of the .tar.gz file extracted into your `~`
+3. Make sure you have the packages nginx-core and php7.4-fpm installed
+4. Show your Nginx running properly
+
+## Hint
+
+If you install Ubuntu 18.04 instead of the Ubuntu 20.04, the version of the php-fpm which will be installed will be the 7.2 instead of 7.4. Then make sure to configure the nginx config file properly.
```

Comment:
Remove "the" between "of" and "Ubuntu 20.04". Remove "the" between "be" and "7.2"